library/std/src/os/windows/net/mod.rs

```diff
@@ -0,0 +1,12 @@
+#![unstable(feature = "windows_unix_domain_sockets", issue = "56533")]
```

Comment:
I think we should cut a new tracking issue following the usual template and close the current issue (#56533) in favor of that new issue.